.devcontainer/README.rst

```diff
@@ -84,9 +34,8 @@ container, you are prompt to enter your license server port and address.
 
 You can set the environment variable :envvar:`ANSYSLMD_LICENSE_FILE`
 from the terminal before launching VS Code.
```

Comment:
```suggestion
You can set the :envvar:`ANSYSLMD_LICENSE_FILE` environment variable 
from the terminal before launching VS Code.
```
Google dev style guide says to follow the code entity with the noun describing what type of object it is.